src/components/atoms/icon.tsx

```diff
@@ -0,0 +1,45 @@
+import { BsPerson } from 'react-icons/bs';
+import { GoHome, GoSearch } from 'react-icons/go';
+import { PiMoon, PiClipboard } from 'react-icons/pi';
+import { IoLanguage } from 'react-icons/io5';
+import {
+  AiOutlineHeart, AiOutlineLike, AiOutlineStar, AiOutlineClose, AiOutlineLeft, AiOutlineBell,
+} from 'react-icons/ai';
+import { HiUserGroup, HiCreditCard } from 'react-icons/hi';
+
+function Icon({ name, size, ariaLabel }:{ name:string; size:string; ariaLabel:string }) {
```

Comment:
```suggestion
interface IconProps {
  name: 'person' | 'home' | 'search';
  size: string;
  ariaLabel: string;
}

function Icon({ name, size, ariaLabel }: IconProps) {
```
TypeScript에서 props의 interface를 다음과 같이 선언하면 컴포넌트를 사용할 때 vscode에서 자동으로 타입 추론을 해 어떤 name을 사용할 수 있는지 알 수 있어서 더욱 편리해요.

Reply:
수정했습니다.